step-01/js/main.js

```diff
@@ -1,26 +1,73 @@
 'use strict';
 
-navigator.getUserMedia = navigator.getUserMedia ||
-    navigator.webkitGetUserMedia || navigator.mozGetUserMedia;
+// Define navigator.mediaDevices.getUserMedia() for older devices.
 
-var constraints = {
+// Old browsers might not implement mediaDevices, so we set an empty object.
+if (navigator.mediaDevices === undefined) navigator.mediaDevices = {};
```

Comment:
Style for these codelabs is that all blocks should be in curly brackets, even if only one line.

Reply:
Acknowledged.